build/SourceLink.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <SourceLinkCreate>true</SourceLinkCreate>
```

Comment:
You the source link to run on whatever you are using as your build server. It doesn't need to run every time someone is doing development locally. It is [setup by default](https://github.com/ctaggart/SourceLink/blob/v2/SourceLink.Create.GitHub/SourceLink.Create.GitHub.targets#L7-L8) to be enabled when the `CI` environment variable is set to `true`. This works for AppVeyor, Travis CI, and others. You can also turn on SourceLinkCreate by passing in the property `dotnet build /p:SourceLinkCreate=ture` or `/p:CI=true`. Hope that helps.